File: brain/8ceb91b6-accb-4798-acac-0df2799833ca/scratch/setup_folders.py
import os

base_path = r"C:\Users\MATHAN\Desktop\A1_Tamil_Primary_Edition\json-db\lessons"
subjects_with_grades = ["english", "maths", "science", "social"]
subjects_flat = ["psychology", "mocktest", "revision", "notes_678"]

# Create subjects with grades and terms
for sub in subjects_with_grades:
    for grade in range(1, 6):
        for term in range(1, 4):
            path = os.path.join(base_path, sub, str(grade), f"Term {term}")
            os.makedirs(path, exist_ok=True)
            print(f"Created: {path}")

# Create flat subjects
for sub in subjects_flat:
    path = os.path.join(base_path, sub, "all")
    os.makedirs(path, exist_ok=True)
    print(f"Created: {path}")

File: brain/8ceb91b6-accb-4798-acac-0df2799833ca/scratch/setup_folders.js
const fs = require('fs');
const path = require('path');

const basePath = "C:\\Users\\MATHAN\\Desktop\\A1_Tamil_Primary_Edition\\json-db\\lessons";
const subjectsWithGrades = ["english", "maths", "science", "social"];
const subjectsFlat = ["psychology", "mocktest", "revision", "notes_678"];

// Create subjects with grades and terms
subjectsWithGrades.forEach(sub => {
    for (let grade = 1; grade <= 5; grade++) {
        for (let term = 1; term <= 3; term++) {
            const dirPath = path.join(basePath, sub, String(grade), `Term ${term}`);
            fs.mkdirSync(dirPath, { recursive: true });
            console.log(`Created: ${dirPath}`);
        }
    }
});

// Create flat subjects
subjectsFlat.forEach(sub => {
    const dirPath = path.join(basePath, sub, "all");
    fs.mkdirSync(dirPath, { recursive: true });
    console.log(`Created: ${dirPath}`);
});
